build_tools/style.fish

```diff
@@ -43,14 +59,22 @@ set -l green (set_color green)
 set -l blue (set_color blue)
 set -l normal (set_color normal)
 
-# Run the fish reformatter if we have any fish files.
 if set -q fish_files[1]
-    if not type -q fish_indent
-        make fish_indent
-        set PATH . $PATH
+    if not command -v fish_indent
```

Comment:
I guess fish_indent from `PATH` is okay for local runs for now
but eventually we want to use `target/debug/fish_indent` or `target/release/fish_indent`
in case we make changes to the formatting.
If `target/debug/fish_indent` is fast enough (5 seconds here, I guess that's fine), can we pass that one explicitly in `build_tools/check.sh` via `PATH=target/debug:$PATH build_tools/style.fish`